server/Dockerfile

```diff
@@ -0,0 +1,19 @@
+from node:10-alpine
+
+maintainer eLife Reviwer Product Team <reviewer-product@elifesciences.org>
+
+workdir /opt
+
+copy package.json .
+copy yarn.lock .
+
+copy dist/ dist/
+
+copy gql/ dist/modules/
+
+copy node_modules/ ./node_modules/
```

Comment:
same reasoning as for `dist/`, should be built within a Docker container
also would probably be populated earlier in the Dockerfile, to take advantage of Docker layer caching